lib/addict/utils.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Addict.Utils do
```

Comment:
This is pretty cool, it's the kind of stuff Phoenix should have in the code base. PR there maybe? :smile: